bundler.d/test.rb

```diff
@@ -25,3 +20,5 @@
   gem 'robottelo_reporter', '~> 0.1'
   gem 'webmock'
 end
+
+gem 'theforeman-rubocop', '~> 0.0.6', groups: [:test, :rubocop]
```

Comment:
From the packaging side https://github.com/theforeman/foreman-packaging/pull/6012 should be merged to deal with the different way of group handling. Note that I wrote that even before I knew it was needed here.

Reply:
oh, right, well we don't need this group for now, so let's just leave it at test :)